pkg/reloadable/tikv.go

```diff
@@ -32,7 +32,7 @@ func CheckTiKV(group *v1alpha1.TiKVGroup, instance *v1alpha1.TiKV) bool {
 	groupTmpl := &group.Spec.Template.Spec
 	instanceTmpl := &instance.Spec.TiKVTemplateSpec
 
-	return equalTiKVTemplate(groupTmpl, instanceTmpl)
+	return equalTiKVTemplate(groupTmpl, instanceTmpl) && !restartAnnotationsChanged(group.Spec.Template.Annotations, instance.GetAnnotations())
```

Comment:
call `restartAnnotationsChanged` in `equalXXXTemplate` func

Reply:
But `TiKVTemplateSpec` does not contain annotations